composer.json

```diff
@@ -14,13 +14,20 @@
 			"email": "jeroendedauw@gmail.com",
 			"homepage": "http://jeroendedauw.com",
 			"role": "Developer"
+		},
+		{
+			"name": "Daniel Kinzler"
 		}
 	],
 	"support": {
 		"irc": "irc://irc.freenode.net/wikidata"
 	},
 	"require": {
-		"php": ">=5.3.0"
+		"php": ">=5.3.0",
+		"wikimedia/assert": "0.2.2"
+	},
+	"require-dev": {
+		"phpunit/phpunit": "3.7.37"
```

Comment:
This is not needed, please remove. That is also a rather out of date version of PHPUnit...